config/routes.rb

```diff
@@ -5,4 +5,11 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  
+  root "users#index"
+
+  get 'users/:user_id/posts' => 'posts#index', as: :user_profile
+  get 'users/:user_id/posts/:id' => 'posts#show', as: :user_profiles
+  get 'users' => 'users#index'
+  get 'users/:id' => 'users#show'
```

Comment:
- [ ] [Optional] Great work implementing the routes for the different controllers and their actions. I would like to suggest a recommended method for setting up routes for controllers. 

```Ruby on Rails
 resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

- Using this method encourages developers to use RESTFUL URL patterns when setting up routes, This concept enables you to manipulate `HTTP` verbs. 👍 
- You can read this [article](https://launchschool.com/books/demystifying_rails/read/routes_and_resources) to have a better understanding of this concept. You can also run `rails routes -c Users` or `rails routes -C Posts` to see the routes created. 👍